spec/acceptance/instance_spec.rb

```diff
@@ -137,6 +138,7 @@ def get_instance(name)
           :project    => 'cloud',
           :created_by => 'aws-acceptance'
         },
+        :tenancy => 'dedicated',
```

Comment:
What requirements adds this to the AWS account the tests are running against? Does that mean the test account needs dedicated instances?